acceptance/acceptance_test.go

```diff
@@ -56,6 +57,7 @@ func init() {
 	flag.BoolVar(&KeepTmp, "keeptmp", false, "Do not delete TMP directory after run")
 	flag.BoolVar(&NoRepl, "norepl", false, "Do not apply any replacements (for debugging)")
 	flag.BoolVar(&Tail, "tail", false, "Log output of script in real time. Use with -v to see the logs: -tail -v")
+	flag.BoolVar(&Force, "force", false, "Force running the specified tests, ignore all reasons to skip")
```

Comment:
Can you name this `-ignore-skip` or `-force-run` or something?

In combination with other flags (e.g. `-update`), this is ambiguous/confusing.

Reply:
done